diff --git a/node_modules/react-digraph/typings/index.d.ts b/node_modules/react-digraph/typings/index.d.ts
index 6b8a616..172e4ac 100644
--- a/node_modules/react-digraph/typings/index.d.ts
+++ b/node_modules/react-digraph/typings/index.d.ts
@@ -96,7 +96,7 @@ declare module 'react-digraph' {
 
   export type SelectionT = { 
     nodes: Map<string, INode> | null, 
-    edges: Map<string, INode> | null 
+    edges: Map<string, IEdge> | null 
   };
 
   export const Edge: React.Component<IEdgeProps>;
